.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+**Reviewer** (@joelea if unsure):
```

Comment:
Still a danger that this becomes a list of people, none of which feel the obligation to move it forward.  Would vote for Primary Reviewer.